packages/table-react/documentation/ExpandableTableExample.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { VFC } from "react";
+import { ExampleComponentProps } from "../../../doc-utils";
+import { Table, TableCaption, TableHead, TableRow, TableHeader, TableBody, TableCell } from "../src";
+import { ExpandableTableRowCell } from "../src/ExpandableTableRowCell";
```

Comment:
De to nye komponentene må eksporteres fra `index.ts`, og gjerne da importeres fra `../src` her i eksempelet.

Reply:
Hupla hops! i fix!